label_studio_ml/examples/tesseract/example.env

```diff
@@ -1,11 +1,11 @@
-LABEL_STUDIO_HOST=http://host.docker.internal:8080
-LABEL_STUDIO_ACCESS_TOKEN=<access token if using local file storage>
+LABEL_STUDIO_HOST=http://10.33.1.37:8089
```

Comment:
please rollback to initial default parameters

Reply:
reverted